pkg/model/health.go

```diff
@@ -0,0 +1,42 @@
+package model
+
+import (
+	"github.com/Merteg/pl-health-service/proto"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+type Health struct {
+	Id         string                `json:"_id,omitempty", bson: "_id, omitempty"`
+	TargetID   string                `json:"targetid,omitempty" validate:"required", bson:"targetid, omitempty" validate:"required"`
+	TargetType string                `json:"targettype,omitempty" validate:"required", bson:"targettype, omitempty" validate:"required"`
+	Status     string                `json:"healthstatus,omitempty",  bson:"healthstatus, omitempty"`
+	Counters   map[string]int32      `json:"counters,omitempty",  bson:"counters, omitempty"`
+	Metrics    map[string]float64    `json:"metrics,omitempty",  bson:"metrics"`
+	Heartbeat  *wrapperspb.BoolValue `json:"heartbeat,omitempty",  bson:"heartbeat, omitempty"`
+	Messages   []*proto.Message      `json:"messages,omitempty", bson:"messages, omitempty"`

```

Comment:
you shouldn't have any proto inside our custom Health struct
Message should be another struct defined here and you should convert proto Messages into custom Messages